src/Components/Home/ListItem.js

```diff
@@ -4,6 +4,10 @@ import { Card, Col, Row, Button, Badge } from 'react-bootstrap'
 function ListItem(prop) {
```

Comment:
We usually call it `props` instead of `prop`.
See: https://reactjs.org/tutorial/tutorial.html#function-components